src/data/ff/camelloneBKS/camelloneBKS.h

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2025 Team Dissolve and contributors
+
+#pragma once
+
+#include "data/ff/ff.h"
+
+// CamelloneBKS Forcefield
+class Forcefield_CamelloneBKS : public Forcefield
+{
+    public:
+    Forcefield_CamelloneBKS() = default;
+    ~Forcefield_CamelloneBKS() override = default;
+
+    // PP override
+    std::vector<PairPotentialOverride> pairPotentialOverrides() override
+    {
+        InteractionPotential<Functions1D> oOPotential(Functions1D::Form::Buckingham128,
+                                                      {133996.2240, 2.76, 16884.9331, 17367.3598, 2315.6480});
+        InteractionPotential<Functions1D> siOPotential(Functions1D::Form::Buckingham128,
+                                                       {1737098.491, 4.8732, 12884.4679, 1929.7066, 578.912});
+
+        std::vector<PairPotentialOverride> overrides{
+            {"O", "O", PairPotentialOverride::PairPotentialOverrideType::Add, oOPotential},
+            {"Si", "O", PairPotentialOverride::PairPotentialOverrideType::Add, siOPotential}};
+
+        return overrides;
+    }
```

Comment:
Yes, basically this, but don't implement this in the header! Also I don't see why you should't be able to use brace-initialisation for the `InteractionPotential` part itself (this will need formatting properly):
```suggestion
    // Return available pair potential overrides
    std::vector<PairPotentialOverride> pairPotentialOverrides() override
    {
       return {
            {"O", "O", PairPotentialOverride::PairPotentialOverrideType::Add, {Functions1D::Form::Buckingham128,
                                                      {133996.2240, 2.76, 16884.9331, 17367.3598, 2315.6480}}},
            {"Si", "O", PairPotentialOverride::PairPotentialOverrideType::Add, {Functions1D::Form::Buckingham128,
                                                       {1737098.491, 4.8732, 12884.4679, 1929.7066, 578.912}}}
                   };
    }
```